lib/wibox/layout/align.lua

```diff
@@ -274,9 +452,8 @@ end
 --
 -- @property expand
 -- @tparam[opt="inside"] string mode How to use unused space.
-
 function align:set_expand(mode)
-    if mode == "none" or mode == "outside" then
+    if mode == "none" or mode == "outside" or mode == "justified" then
```

Comment:
That makes a lot of possible modes. I think something functional like this would be more readable:

```lua
if gears.table.has_item({
    "none",
    "outside",
    "justified",
}, mode) then
```

(bonus point with the "enum like table" to handles modes: this test would be always up -to-date)

Reply:
True, now it is like this after [b37ba4e](https://github.com/awesomeWM/awesome/pull/3588/commits/b37ba4e4a54266187492729c2d42691c54ffddc2)